java/kotlin-extractor/src/main/java/com/semmle/extractor/java/OdasaOutput.java

```diff
@@ -547,6 +547,16 @@ public boolean equals(Object obj) {
 			}
 		}
 
+        @Override
```

Comment:
Looks like there's an indentation mismatch with the surrounding code; probably spaces vs tabs?

Reply:
might also just be me lazily relying on autoformat on save or on commit for the files I typically edit, not having that for java files :slightly_smiling_face: